.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install
-    - run: npm start & sleep 5 && npm test
+    - run: npm test
```

Comment:
<img width="1028" alt="image" src="https://user-images.githubusercontent.com/38065883/172470590-746390cd-c8e7-47fd-ac7b-348a6bc1d093.png">
add `cursor: pointer` to the form button